package.json

```diff
@@ -61,7 +61,7 @@
     "mine": "node ./apps/shared/test-helpers/mine.js",
     "postinstall": "npm run bootstrap",
     "publish:apps": "lerna run publish",
-    "publish:http": "lerna run --scope='@tps/apps-*' publish:http --stream",
+    "publish:http": "lerna run --scope='@tps/apps-*' publish:http --stream --parallel",
```

Comment:
I would probably try to remove concurrency setting from `lerna.json` config file instead, since lerna by default works in parallel, that config setting was specifically added for this npm script (you may quickly check this by searching "lerna run" recursively and find that there are more `--parallel` uses and just a single `--concurrency 1` use case